cmd/pixiu/pixiu.go

```diff
@@ -40,7 +40,7 @@ import (
 )
 
 // Version pixiu version
-var Version = "0.1.0"
+var Version = "0.2.0"
```

Comment:
0.3.0?

Reply:
maybe v0.2.1 can merge this PR

Reply:
@xiaoliu10  Sorry for my bothering, but I've sent something about [ospp](https://summer.iscas.ac.cn/#/org/orgdetail/dubbogo) program to your email, and there is also an issue I've created about that. Could you please take a look? Thank you so much! :blush: